libs/dal/src/repositories/job/job.repository.ts

```diff
@@ -207,6 +208,27 @@ export class JobRepository extends BaseRepository<JobDBModel, JobEntity, Enforce
       }
     }
 
+    if (filtered) {
```

Comment:
If filtered, we want a new digest to be created, and not merged to an existing delayed digest